distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2301,6 +2301,18 @@ Event event = new Event("ddf/notification/catalog/downloads", properties);
 eventAdmin.postEvent(event);
 ----
 
+=== Configuring system.properties
+
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through system properties at the very beginning of Karaf's boot process. This file can be used to change specific system-level properties such as thread-pool count and default directory locations.
+
+==== Thread Pool
+
```

Comment:
We may want to say something about UI searches too since we're using this value to configure the thread pool used by the Cometd endpoint/UI.